wherehows-backend/app/controllers/Application.java

```diff
@@ -76,4 +81,37 @@ public static Result index() {
   public static Result healthcheck() {
     return ok("GOOD");
   }
+
+  public static Result deploylibsbackend() {
+    String env = System.getenv("WHZ_APP_HOME");
```

Comment:
I have made it as private static and declared before and used in the method. I haven't made the variable as final because i need to modify the value to get into sub-directory